### ISSUE TITLE
New list of 49 auto-antibodies

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#6990]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

Dear all,
Please find attached a new list with 49 new autoantibodies.
Many thanks,
Maud

[20210517 BrainTeam VF.xlsx](https://github.com/obophenotype/human-phenotype-ontology/files/6554953/20210517.BrainTeam.VF.xlsx)

**Comment by pnrobinson** [comment]
@matentzn  Could we write a script for these new terms?

**Comment by matentzn** [comment]
Great, yeah:

So this is the reformatted table you can work with: https://docs.google.com/spreadsheets/d/1JI5q4rHEMFWYMliSXe7Dsz051nAbkMllSI8TaB6ydLY/edit#gid=0

The pipeline is [here](https://github.com/obophenotype/human-phenotype-ontology/pull/6997).

I would recommend we make sure the antibody table is perfect before merging. TODOs:

- [ ] Review the definition (I used a google formula to generate it: basically your string and some pruning on the label) and make sure you are happy.
- [ ] Make sure the parents of HP:5000022 and HP:5000023 are correct
- [ ] Tell me what to do with the `Diseases characterized by this term ? (e.g. Orphanet or OMIM number)` column

**Comment by matentzn** [comment]
@MaudDLVT there is an issue with the one of your antibody terms: it appears duplicate, but it has different papers attached with it.. could you check the spreadsheet and advice?

**Comment by matentzn** [comment]
@MaudDLVT a second thing - we are currently not set up for processing your "disease is characterised by" data, so will ignore these unfortunately. We leave it up to you however - if you provide us with OMIM ids instead (no ORPHA, not labels), we can ingest these as HPO annotations. We leave this up to you, but feedback appreciated so we know we are agreed.

**Comment by MaudDLVT** [comment]
@matentzn Thanks for the checked. You are right, one ligne is duplicated. Should I have to group the two lines (17 and 43)?

**Comment by matentzn** [comment]
Feel free - this is your data!

**Comment by MaudDLVT** [comment]
@matentzn It's quite important for the MD to have the correspondance with the ORPHA codes. Is this possible to have ORPHA on HPO database? (e.g.: You have ORPHA association in HP:0030057)

**Comment by matentzn** [comment]
I understand - we are just not set up for ORPHA codes at the moment. It would take months or longer to get your data in. At the moment, we can only support OMIM codes, but generally, you should be able to find a mapping from the OMIM codes to the Orpha codes - ORDO itself I think has a quite comprehensive set of mappings.. We may be able to convince @pnrobinson  to add the codes as "comments" to the class, but I doubt it.

So you can discuss this with your team:
1. We can ingest OMIMs
2. you can use your Orphanet to Omim mappings to translate the mappings in your database

Or you try again in a 6 months, perhaps we are set up then!

**Comment by matentzn** [comment]
Hahahha :) You are funny... Ok, lets try a bit later in the year then. In any case, these codes (neither OMIM nor Orpha) go into HP.owl, so we can go ahead with adding the terms and figuring the disease stuff out later. Let me know when you have deleted the duplicate row in the spreadsheet.

**Comment by drseb** [comment]
What exactly do you want to state with your reference to Orphanet? Do you want to express that an HPO term is equivalent to an Ordo-code? Or just cross-reference them?

**Comment by MaudDLVT** [comment]
I am done with the Anti-GlyR antibody. I will have a third upload this summer (this one is the second) with more autoantibodies.

**Comment by drseb** [comment]
Because if you want to state that an ORDO-disease is characterised by an HPO term, orphanet is doing this and creating these associations. We already  ingest those and make them available in our hpo-annotation downloads

**Comment by MaudDLVT** [comment]
@drseb So, please correct me if I am wrong, the disease associations displayed on the HPO website is coming from ORDO or Orphadata and not directly from you or contributors?
The goal of this extensive research on autoantibodies is to help MD for coding patients with a broad disorder (e.g. Autoimmune encephalitis) + an HPO term for autoantibody to be more specific on the database. Thus, the disease association is quite important to help the MD with his search on HPO website.
Please tell me if I am not clear enough.

**Comment by drseb** [comment]
The disease associations come from multiple resources, e.g. OMIM, Orphanet, and manual curation of the HPO team and collaborators. 
see for example: https://hpo.jax.org/app/browse/disease/ORPHA:915
And
https://hpo.jax.org/app/browse/disease/OMIM:100050

**Comment by MaudDLVT** [comment]
Could you considered my table as a manual curation from collaborators? It has been reviewed by French MDs.

**Comment by drseb** [comment]
Process-wise, my personal opinion would be to make those annotations on the Orphanet side, but we can ingest this as well. @pnrobinson to decide! Or @nicolevasilevsky

**Comment by pnrobinson** [comment]
For the most part the affected diseases are non-Mendelian rare and thus we do not annotate them. I think it is best to leave this to orphanet!

**Comment by matentzn** [comment]
@pnrobinson I think the PR is ready now, please do a last sanity check of the [diff](https://github.com/obophenotype/human-phenotype-ontology/pull/6997/files?file-filters%5B%5D=.owl), then merge into master and edit to your hearts content.

PS: Contact me if you want me to streamline this process (an external contributor adding a number of terms at once) in the future - I can write up instructions for the term supplier, and make it even easier for us.
